pilot/pkg/features/pilot.go

```diff
@@ -298,6 +298,20 @@ var (
 			"Istio Resources",
 	).Get()
 
+	AnalysisInterval = func() time.Duration {
+		val, _ := env.RegisterDurationVar(
+			"PILOT_ANALYSIS_INTERVAL",
+			10000*time.Millisecond,
+			"If analysis is enabled, pilot will run istio analyzers using this value as interval in seconds "+
+				"Istio Resources",
+		).Lookup()
+		if val == 0 {
```

Comment:
What if the duration is set less than 1 second but more than 0?

Reply:
I was wondering if 0.5 second is an ok value, not really sure. what do you think?